macros/create_pipe.sql

```diff
@@ -53,6 +63,20 @@ copy into {{ schema_name }}.{{ table_name }} from (
     )
 {% endset %}
 
+{% set aws_ingest_copy_statement %}
```

Comment:
wondering if we can move this back into one copy statement by looking at if MATCH_BY_COLUMN_NAME or ERROR_ON_COLUMN_COUNT_MISMATCH is present in the config.

```sql
{% if 'match_by_column_name' in copy_options %}
copy into {{ schema_name }}.{{ table_name }}
from @{{ schema_name }}.{{ table_name }}_stage
{% else %}
copy into {{ schema_name }}.{{ table_name }} from (
  select
  ...
{% endif %}
on_error = continue
```